ui/src/app/applications/components/application-parameters/application-parameters.tsx

```diff
@@ -130,6 +133,12 @@ export const ApplicationParameters = (props: {
     const [removedOverrides, setRemovedOverrides] = React.useState(new Array<boolean>());
 
     let attributes: EditablePanelItem[] = [];
+    let appValues: string;
+    if (source && source.helm && source.helm.values) {
+        isValuesRaw = typeof source.helm.values !== 'string'; // nolint
```

Comment:
I'm feeling a bit dense.... shouldn't this logic toggle between `source.helm.values` vs `source.helm.valuesObject`? Or does the server return the preferred option on the same `.helm` key?

Reply:
I'll trust the process.

Reply:
You're completely right @crenshaw-dev! 🙈  That is indeed an error; I've a fix for it in #14257.